### PR TITLE
Revert Prow bump and rules_nodejs bump.

### PR DESCRIPTION
**Pull request by cjwagner**
See this issue for details: https://github.com/kubernetes/test-infra/issues/12852

TLDR; The `/pr` endpoint is broken and `/plugins` and `/command-help` are experiencing problems due to an apparent missing dependency problem. It is likely that this problem was introduced in https://github.com/kubernetes/test-infra/pull/12806 so we are reverting that and [the most recent Prow bump](https://github.com/kubernetes/test-infra/pull/12813) that deployed it.

Reverted commits are
- [270a6824e0105ce6f40609ea4b845fe625d78872](https://github.com/kubernetes/test-infra/pull/12806/commits/270a6824e0105ce6f40609ea4b845fe625d78872)
- [e0844969a43202587e0c0883932797403a3704d5](https://github.com/kubernetes/test-infra/pull/12813/commits/e0844969a43202587e0c0883932797403a3704d5)

/cc @tony-yang @stevekuznetsov @Katharine @fejta

### CODE REVIEW

**Comment by fejta** [comment]
I would just like to rollback the deployment in this PR.

I suspect we need to take the rules_nodejs upgrade

**Comment by Katharine** [comment]
I'm not sure whether failing to build is actually worse than building incorrectly, though I concede the blast radius of the incorrect builds is smaller. I think it makes sense to revert both for the time being so people don't make bad builds? I don't feel very strongly about it though given the trade-off of totally busted builds on latest bazel is pretty bad too.

I intend to fix it properly tomorrow either way.

**Comment by fejta** [comment]
Sure SGTM. Primarily concerned with ensuring we prioritize getting things working with the rules_nodejs upgrade

**Comment by cjwagner** [comment]
The revert seems to have resolved the bugs described in https://github.com/kubernetes/test-infra/issues/12852
(Notably https://prow.k8s.io/pr is working again.)
